pkg/driver/docker/docker.go

```diff
@@ -152,6 +153,16 @@ func (d *dockerDriver) RunDevContainer(
 	if mergedConfig.Privileged != nil && *mergedConfig.Privileged {
 		args = append(args, "--privileged")
 	}
+
+	// In case we're using podman, let's use userns to keep
+	// the ID of the user (vscode) inside the container as
+	// the same of the external user.
+	// This will avoid problems of mismatching chowns on the
+	// project files.
+	if d.Docker.DockerCommand == "podman" {
```

Comment:
What happens if the user uses podman as an alias to docker? Is there a way we can find out if the docker command itself is podman?